SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -169,8 +180,15 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 			{
 				ClientRequirementSet = *RpcRequirementSet;
 			}
+
 			TArray<Worker_ComponentId> ComponentIds;
 			Acl->ComponentWriteAcl.GetKeys(ComponentIds);
+			// Ensure that every component ID in ComponentPresence is set in the write ACL.
+			for (const auto& RequiredComponentId : ComponentPresenceComponent->ActorComponentList)
+			{
+				ComponentIds.AddUnique(RequiredComponentId);
```

Comment:
although actually it might make sense to put every component in the ACL as it simplifies this logic, and I have no idea if it would actually be a performance problem